daemon/daemon.go

```diff
@@ -1013,8 +1013,8 @@ func (daemon *Daemon) changes(container *container.Container) ([]archive.Change,
 	return container.RWLayer.Changes()
 }
 
-// TagImage creates a tag in the repository reponame, pointing to the image named
-// imageName.
+// TagImage creates a tag in the repository and with the name specified by newTag,
```

Comment:
I'm still finding this confusing. How about something like "TagImage creates the tag specified by newTag"?